cmd/lambda/leases/list.go

```diff
@@ -37,18 +38,17 @@ func GetLeases(w http.ResponseWriter, r *http.Request) {
 		leaseResponseItems = append(leaseResponseItems, response.LeaseResponse(*lease))
 	}
 
-	if err != nil {
```

Comment:
error is already handled on line 30
https://github.com/Optum/dce/pull/194/files#diff-3d32712d4ead5725d017ab19e54d8a14R30